lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -369,6 +373,30 @@ spec = do
             , expectErrorMessage errMsg403NonNullReward
             ]
 
+    it "STAKE_POOLS_JOIN_05 - Can join when stake key already exists" $ \ctx -> do
+        let (walletWithPreRegKey:: [Text]) =
+                [ "over", "decorate", "flock", "badge", "beauty"
+                , "stamp" , "chest", "owner", "excess", "omit"
+                , "bid", "raccoon", "spin" , "reduce", "rival"
+                ]
+        let payload = Json [json| {
+                "name": "Wallet with pre-registered stake key",
+                "mnemonic_sentence": #{walletWithPreRegKey},
+                "passphrase": "Secure Passphrase"
+                } |]
+
+        threadDelay 4000000
```

Comment:
Why the delay?

Reply:
We try to join immediately when the wallet is created. It has no time to find the existing key reg cert.

Should probably use `eventually` though. Just sprinkled it in to get it passing.